e2e/optimism-stack.Dockerfile

```diff
@@ -23,24 +23,40 @@ RUN apt-get install -y jq nodejs npm netcat-openbsd
 
 RUN npm install -g pnpm
 
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+ENV PATH="${PATH}:/root/.cargo/bin"
+
+WORKDIR /git
+RUN git clone https://github.com/casey/just
+WORKDIR /git/just
+RUN cargo install just
+
+WORKDIR /git
+
+RUN curl -L https://foundry.paradigm.xyz | bash
+
+RUN . /root/.bashrc
+
+ENV PATH="${PATH}:/root/.foundry/bin"
+
+RUN foundryup
+
+RUN forge --help
+
 WORKDIR /git
 COPY --from=build_1 /git/op-geth /git/op-geth
 WORKDIR /git
 RUN git clone https://github.com/hemilabs/optimism
 WORKDIR /git/optimism
+RUN echo 6666666ddd
```

Comment:
Was this for debugging?

Reply:
yeah It was to break the docker build cache to pull in new commits as I was testing, I'll remove